File: app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;


import android.os.Bundle;

import com.example.myapplication.fragment.MainFragment;

public class MainActivity extends BaseFragmentActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        switchFragment(MainFragment.newInstance());
        setToolbarTitle(R.string.title_main);
    }

    @Override
    protected int getLayout() {
        return R.layout.activity_main;
    }
}

File: app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.os.Bundle
import com.example.myapplication.fragment.MainFragment

class MainActivity : BaseFragmentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        switchFragment(MainFragment.newInstance())
        setToolbarTitle(R.string.title_main)
    }

    override fun getLayout(): Int {
        return R.layout.activity_main
    }
}